build.gradle

```diff
@@ -3,15 +3,15 @@
 buildscript {
 
     ext {
-        kotlin_version = '1.8.10'
+        kotlin_version = '1.9.20'
         spotless = '6.1.2'
-        anvil_version = '2.4.4'
-        ksp_version = '1.8.10-1.0.9'
+        anvil_version = '2.4.8'
+        ksp_version = '1.9.20-1.0.14'
         gradle_plugin = '8.1.2' // When updating, also update lint_version
         lint_version = '31.1.2' // This value must always be gradle_plugin + 23
         min_sdk = 23
-        target_sdk = 33
-        compile_sdk = 33
+        target_sdk = 34
```

Comment:
are we sure we want to change `target_sdk` in here? have you confirmed if this will bring any new behaviours and if they are all desired? 

If not, we presumably can defer changing the target sdk while still updating everything else?

Reply:
Changed to 33 ✅